src/Symfony/Component/VarDumper/Tests/Dumper/ContextProvider/RequestContextProviderTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dumper\ContextProvider;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Dumper\ContextProvider\RequestContextProvider;
+
+class RequestContextProviderTest extends TestCase
+{
+    public function testGetContextOnNullRequest()
+    {
+        if (!class_exists(RequestStack::class)) {
```

Comment:
`@requires function \Symfony\Component\HttpFoundation\RequestStack::__construct` on the class instead of those

Reply:
I was hesitating on using the `__construct` to check class existence. Will update :+1:

Reply:
Updated, as well as other occurrences!